src/main/java/uk/ac/ebi/embl/flatfile/writer/genbank/ContigWriter.java

```diff
@@ -43,7 +43,7 @@ public boolean write(Writer writer) throws IOException {
       } else {
         firstContig = false;
       }
-      FeatureLocationWriter.renderLocation(block, contig, false, false);
+      FeatureLocationWriter.renderLocation(block, contig);
```

Comment:
We never want to write partiality for Contig lines. The old code prevented this completely. Now we will write partiality if the location has it.